adafruit_adxl34x.py

```diff
@@ -33,49 +33,57 @@
 from micropython import const
 from adafruit_bus_device import i2c_device
 
-__version__ = "0.0.0-auto.0"
-__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_ADXL34x.git"
-_ADXL345_DEFAULT_ADDRESS = const(0x53)  # Assumes ALT address pin low
+try:
+    from typing import Tuple, Dict
+
+    # This is only needed for typing
+    import busio  # pylint: disable=unused-import
+except ImportError:
+    pass
+
+__version__: str = "0.0.0-auto.0"
```

Comment:
i think dont type the __'s im not sure it wont break our packaging setup

Reply:
Ah! I hadn't thought about that. Apparently there is some nuance to the typing mechanism and how it interacts with things that I don't fully understand. Adjusting now. Thanks!